test/general/test_patches.py

```diff
@@ -0,0 +1,11 @@
+﻿import unittest
+from worlds.AutoWorld import AutoWorldRegister
+from worlds.Files import AutoPatchRegister
+
+
+class TestPatches(unittest.TestCase):
+    def test_patch_name_matches_game(self) -> None:
+        for game_name in AutoPatchRegister.patch_types:
+            with self.subTest(game=game_name):
+                self.assertIn(game_name, AutoWorldRegister.world_types, 
+                                f"Patch '{game_name}' does not match the name of any world.")
```

Comment:
both formatting suggestions (indenting and the newline at the end of the doc)
and using `.keys()` to make the error message more concise
```suggestion
                self.assertIn(game_name, AutoWorldRegister.world_types.keys(),
                              f"Patch '{game_name}' does not match the name of any world.")

```